src/dialogs/symlink.tsx

```diff
@@ -0,0 +1,274 @@
+                    onClick={() => {
+                        const filename_invalid = checkLinkName(symlinkName, cwdInfo?.entries || {}, selected);
+                        if (filename_invalid) {
+                            setSymlinkError(filename_invalid);
+                        } else {
+                            createLink();
+                        }
+                    }}
+                    isDisabled={errorMessage !== undefined ||
+                        symlinkError !== null ||
+                        cwdInfo === null ||
+                        user === null}
+                  >
+                      {_("Create link")}
+                  </Button>
+                  <Button variant="link" onClick={() => dialogResult.resolve()}>{_("Cancel")}</Button>
+              </>
+          }
+        >
+            {errorMessage &&
+            <InlineNotification
+              type="danger"
+              text={errorMessage}
+              isInline
+            />}
+            <Form
+              isHorizontal
+              onSubmit={e => {
+                  e.preventDefault();
+                  createLink();
+                  return false;
+              }}
+            >
+                <FormGroup fieldId="target-name" label={_("Target")}>
+                    <TextInput
+                      value={selected.name}
+                      readOnlyVariant="plain"
+                      id="target-name"
+                    />
+                </FormGroup>
+                <FormGroup fieldId="symlink-input" label={_("Symlink name")}>
+                    <TextInput
+                      validated={symlinkError ? "error" : "default"}
+                      value={symlinkName}
+                      onChange={(_, val) => {
+                          const filename_invalid = checkLinkName(val, cwdInfo?.entries || {}, selected);
+                          setSymlinkError(filename_invalid);
+                          setErrorMessage(undefined);
+                          setSymlinkName(val);
+                      }}
+                      onFocus={(event) => event.target.setSelectionRange(0, cockpit.format(_("link to $0"), "").length)}
+                      id="symlink-input" autoFocus // eslint-disable-line jsx-a11y/no-autofocus
+                    />
+                    <FormHelper fieldId="symlink-input" helperTextInvalid={symlinkError} />
+                </FormGroup>
+                <FormGroup
+                  fieldId="symlink-type"
+                  label={_("Type")}
+                  isInline
+                  labelIcon={
+                      <Popover
+                        headerContent={_("Absolute vs. Relative")}
+                        bodyContent={
+                            <>
+                                {/* eslint-disable-next-line max-len */}
+                                <p>{_("Absolute symlinks are ideal when referring to files or directories that won't move, such as system files..")}</p>
```

Comment:
This should end either with a one dot or three dots

Reply:
One dot suffices I think :+1: